src/connectionhealthpolicy/ConnectionHealthPolicyConfiguration.ts

```diff
@@ -24,4 +24,6 @@ export default class ConnectionHealthPolicyConfiguration {
   missedPongsUpperThreshold = 4;
   maximumAudioDelayMs = 60000;
   maximumAudioDelayDataPoints = 10;
+  sendingAudioFailureSamplesToConsider = 2;
```

Comment:
Nit: I know *none* of the other members do this, but can we comment why we chose these values here or in the guide?

Reply:
Thanks for reminding, I wanted to add that info. I'm inclined to adding it to the description of the PR so it's easily visible

Reply:
I think adding a comment here would be helpful over PR description as once this is merged its extra steps to get to the PR :)

We do that in [`MeetingSessionConfiguration`](https://github.com/aws/amazon-chime-sdk-js/blob/main/src/meetingsession/MeetingSessionConfiguration.ts).

Reply:
Makes sense but unlike [MeetingSessionConfiguration](https://github.com/aws/amazon-chime-sdk-js/blob/main/src/meetingsession/MeetingSessionConfiguration.ts) where we documented the meaning of each property, what Henry was asking for is the rationale for choosing the value (in this case 2) which in this case is a bit verbose.

The rationale here is "2000 ms is the upper bound for median (p50) of the `attendeePresenceDurationMs` values across all meetings using event ingestion and reporting the `attendeePresenceReceived` event."  I feel this would be better off inside the guide rather than the code. Let me know if you disagree

Reply:
No strong opinions here either way is fine for me.